service/initialise.go

```diff
@@ -24,7 +26,15 @@ func NewServiceList(initialiser Initialiser) *ExternalServiceList {
 }
 
 // Init implements the Initialiser interface to initialise dependencies
-type Init struct{}
+type Init struct {
+	CantabularClientFactory func(cfg cantabular.Config, ua dphttp.Clienter) *cantabular.Client
```

Comment:
It's a test seam for unit testing. There may be better ways. Bascially I wanted to be able to inject a spy here instead of the default implementation?

Reply:
Cool.